climpred/temperature_profile_calculation.py

```diff
@@ -63,4 +63,4 @@ def calculate_temperature_matrix():
     epsilon_inverse_matrix = np.linalg.inv(epsilon_matrix)
 
     temperature_matrix = (np.dot(epsilon_inverse_matrix, rhs_vector))**0.25
-    return np.around(temperature_matrix, 2)
+    return np.around(temperature_matrix - 273.15, 2)
```

Comment:
Add comment to specify that you're converting to celsius, and that we'll add functionality to change later